data/climateDat.csv

```diff
@@ -0,0 +1,57 @@
+Year,annual_T,annual_CO2
```

Comment:
I don't think this file should be checked in since it gets generated. It should be generated when people run scripts but doesn't need to be part of the repository. That's because, in the future when we use different data, we don't want to have to make a PR and confirm these changes manually. We will confirm the code that generates it.